### ISSUE TITLE
Option to have areas proportional to the *log* of the populations?

### ISSUE THREAD

**Issue by seedjay1** [jolars/eulerr#96]
Sweet package - thanks!

One issue I'm having: some populations in my data are much greater than others (like 10^6 vs 10^2), which results in the smaller ellipses being "smushed" together with label text overlapping etc. 

I'm thinking if ellipse areas could be set to be proportional to the logarithm of the category populations, that issue would be mitigated somewhat.

Thoughts on effectiveness/feasibility?

**Comment by jolars** [comment]
I can see a use case for this. The big question is probably whether the transformation should live on the input side, and thus affect the actual fit too, or just be a plotting scale. Thoughts?

**Comment by gwselke** [comment]
Not sure whether I understand the difference between "actual fit" and "plotting scale".
I would have thought that any transform would affect all of what eulerr sees, thinks and does, *except* any labels shown in the diagram.

**Comment by jolars** [comment]
Yes, on second thought you're absolutely right. It wouldn't make any sense to try to make this a plot-only option.

**Comment by jolars** [comment]
You can now achieve this with `transform = log` or `transform = log1p` or whatever you like. It applies to the exclusive regions.
